ee/agent/tests/informers_stacks_test.go

```diff
@@ -43,21 +48,62 @@ var _ = Describe("Stacks informer", func() {
 				}).
 				Do(context.Background()).
 				Into(stack)).To(Succeed())
-
-			stack.Status.Ready = true
+		})
+		It("Should be disabled and have sent a Status_Progressing", func() {
 			Expect(
-				k8sClient.Put().
+				k8sClient.Patch(types.MergePatchType).
```

Comment:
Here, if you use .Into(stack), the client will populate the object with the response from the kube api server, avoiding the need to get the stack later